trunk/examples/concurrent/bpl/regression/skip.yml

```diff
@@ -0,0 +1,7 @@
+# Timeout with Automizer, GemCutter succeeds
+TIMEOUT:
+- MirabelleConcurrentIncrement.bpl ReachSafety-32bit-Automizer.epf ReachSafety.xml
```

Comment:
A slightly strange mix of YAML and our own syntax. Why not
```yml
- program: MirabelleConcurrentIncrement.bpl
  settings: ReachSafety-32bit-Automizer.epf
  toolchain: ReachSafety.xml
```
?

Reply:
Good point, this makes things much easier 🙂
I just adapted this. One minor issue is that this makes the readability slightly worse in my opinion.